### DISCUSSION TITLE
Bet not listed in Cloudbet website when using Cloudbet Trading API

### DISCUSSION THREAD

**Discussion by jbn1981** [cloudbet/docs#74] [Q&A]
Hi,

I placed a bet using Cloudbet Trading API endpoint **/v3/bets/place**.

It worked as I can see it when I call Cloudbet Trading API endpoint **/v3/bets/history**. In fact, Cloudbet support team told me it was a valid bet cause they could see it when I told them to check it (giving them the bet referenceId).

The problem is that If I log into the Cloudbet website I can't see my bet listed under Settled Bets. I can see the ones made in the website but not the ones made with the API.

Cloudbet support tell me to ask here as they can't give a solution.

Anybody?

Thanks.

**Comment by kgravenreuth** [comment]
Hi @jbn1981 ,

Thanks for highlighting this! We're considering addition of bets placed with the Trading API to our Cloudbet website's Settled Bets view, and will let you know if and when this is implemented.

Please let me know if you have any other questions.

Cheers,
Klaus Gravenreuth
Product & Engineering @ Cloudbet

**Comment by kgravenreuth** [comment]
Hi @jbn1981 
Good news, bets placed with the Cloudbet Trading API should now be available in our Cloudbet website's Settled Bets view once settled.

Please let me know if you have any other questions.

Cheers,
Klaus Gravenreuth
Product & Engineering @ Cloudbet

**Reply by jbn1981** [reply]
Awesome! I saw it days ago and it worked but... is it working right now? I can't see any settled bets in this moment, none at all.

**Reply by kgravenreuth** [reply]
Hi @jbn1981 ,

The settled bets, including bets placed using the Cloudbet Trading API, should be available on your Full Bet History at https://www.cloudbet.com/en/sports/bets?tab=settled

Could you please check again?

![settled_bets](https://github.com/user-attachments/assets/ce4110f6-f705-4e06-b465-db6fd6af3023)

Cheers,
Klaus Gravenreuth
Product & Engineering @ Cloudbet

**Reply by jbn1981** [reply]
Yes, now it's working.

You're doing a great job, thank you.

**Reply by cuzmuc** [reply]
Hi, @jbn1981!
I hope you're doing well.
I have much trouble with programmatic betting,
I want you to help me a lot for fixing my project, and to discuss with you about betting in detail.
This is my email:  cuzmucghennadii@gmail.com
Looking forward to good news from you!
Best regards,
Cuzmuc G

**Reply by skobbejakkals** [reply]
Hi @kgravenreuth, should open bets placed through the API also be visible on the website? I can see my API settled bets there but not open bets.

**Reply by kgravenreuth** [reply]
Hi @skobbejakkals ,
You can also view the open bets placed via the Cloudbet API on the `List by Date` tab within the `Open Bets` page. Note that these open bets are only visible on the `List by Date` tab but not on the `Group by event` tab. 

[Bet History page](https://www.cloudbet.com/en/sports/bets?urls.primaryName=Trading&utm_source=github.com&utm_medium=community&utm_campaign=axis)


![List_by_date_open_bets](https://github.com/user-attachments/assets/552ba6b8-43f3-439b-b859-8bcf1207d2a3)

Cheers,
Klaus Gravenreuth
Product & Engineering @ Cloudbet

**Reply by skobbejakkals** [reply]
Hi @kgravenreuth,

Thanks for getting back to me. I just checked the "List by date" tab under Open bets and nothing is listed. I have multiple bets in play through the API.

**Reply by kgravenreuth** [reply]
Hi @skobbejakkals ,

Could you please show us a screenshot with the contents of the `List by date` tab if you don't mind? Please note that you may have to refresh the page for newly placed bets to show up.

Cheers,
Klaus Gravenreuth
Product & Engineering @ Cloudbet

**Reply by skobbejakkals** [reply]
Hi @kgravenreuth,

I found some more info. The API bets only show under "List by date" if you **also have website placed bets**. If all my bets are API bets the "List by date" tab is always empty.

Screenshot below is with 3 In-Play active API bets.

![image](https://github.com/user-attachments/assets/f6773fb6-bff7-4b00-8078-9b35354c2701)

As soon as I place a webiste bet (top bet), all 4 my bets show up

![image](https://github.com/user-attachments/assets/faf75cbb-5db4-4e62-b1de-0ecd9be5cfc9)

**Reply by kgravenreuth** [reply]
Thanks for the details @skobbejakkals . We'll check and get back to you on this. Glad that you could figure out a way to display the API bets for now

**Reply by vendreel** [reply]
Hello, i still have this issue, i cannot find where my bets are after placing via API
